### DISCUSSION TITLE
Every feed I add comes up with "Access Denied"

### DISCUSSION THREAD

**Discussion by sfc-juventino** [sebdelsol/sfvip-all#15] [General]
I know the credentials are correct because they work in other players. I have tried 2 different ones. Any ideas ?

**Comment by sebdelsol** [comment]
Please share your [logs](https://github.com/sebdelsol/sfvip-all?tab=readme#logs)

**Comment by sfc-juventino** [comment]
Please find attached. The Stop lines are probably after I get the Access
Denied message and close the application. I have checked the firewall and
the application has access.

Thanks for your time and help.

**Comment by sebdelsol** [comment]
I don't see the logs

**Reply by sfc-juventino** [reply]
[Logs.zip](https://github.com/sebdelsol/sfvip-all/files/14474182/Logs.zip)
Apologies. Not sure what went wrong there.

**Reply by sebdelsol** [reply]
What's weird is that I see those 2 lines in the logs:
>2024-03-03 11:26:52,719 - Inject 'All Movies' category (id=2656) for 'get_vod_categories' request
2024-03-03 11:26:53,957 - Inject 'All Series' category (id=2657) for 'get_series_categories' request

It means it has accessed the Xtream account but failed after injecting the all categories...Have you tried launching this account on Sfvip Player ? 

EDIT: you could try to increase the `network timeout` in the settings:
![image](https://github.com/sebdelsol/sfvip-all/assets/6039281/2b3bad44-f6d6-47e6-a892-d1d6ea2be342)

**Reply by sfc-juventino** [reply]
![image](https://github.com/sebdelsol/sfvip-all/assets/102561215/4f2403a1-978a-4fd9-b09b-73fc990126fd)
Increased the maximum of 600. But it fails way before 60 seconds anyway.

This is now on a different PC but the result is the same. Again, working on another player.

**Reply by sebdelsol** [reply]
Have you tried on Sfvip player (not sfvip-all) ?

**Reply by sfc-juventino** [reply]
I thought that I was trying SFVIP Player ...am I not ?

**Reply by sebdelsol** [reply]
It's Sfvip-all which wraps Sfvip player...

**Comment by sfc-juventino** [comment]
I have downloaded just SFVIP player from here :
![image](https://github.com/sebdelsol/sfvip-all/assets/102561215/f75c4df5-a45a-4ca8-a406-83dd33131810)
But I get the same result, unfortunately.

**Reply by sebdelsol** [reply]
Is this an xtream account ? And if so is there any way to test it ?

**Reply by sfc-juventino** [reply]
Its not, no.

But I've just tried this:
https://iptv-org.github.io/iptv/index.m3u
and it works. So not sure what is going on

**Reply by sfc-juventino** [reply]
A friend of mine has just come over and tried his xtreme account link and still no deal, not in this and not in the SFVIP player

The format is and works in other IPTV Player: 
http://servername.org/get.php?username=XXXXXXXX&password=YYYYYYYYY&type=m3u_plus

Always the same Access Denied error

**Reply by sebdelsol** [reply]
There are 3 types of account : m3u, Xtream and MAC.
For Xtream & Mac the url is not supposed to have arguments: there are fields to fill up for that in the the interface.
If it's an m3u url then it's supposed to directly point to the m3u8 file.

**Comment by sfc-juventino** [comment]
The problem is that even if I use the username, password and server URL
method, it still fails with the same error message.

**Reply by sebdelsol** [reply]
I've tried 2 services that had m3u playlist with username & password. 
None worked with the m3u url suggested (which was in the exact same form has the one you have)
I tried in my browser, VLC and a lot of other apps to no avail.
By chatting with their customers services both  told me I had to use the Xtream login instead and it worked.

Choose:
![image](https://github.com/sebdelsol/sfvip-all/assets/6039281/a471b897-48ff-4ad9-8352-343de947a6de)

if your url is **http<span>://<span>servername.org**/get.php?username=**XXXXXXXX**&password=**YYYYYYYYY**&type=m3u_plus
Then fill up the required fields this way:
* Address:  http://servername.org
* Username: XXXXXXXX
* Password: YYYYYYYYY

**Reply by sfc-juventino** [reply]
Yes, I have done this  in this format :
Address: http://servername.org/
Username: XXXXXXXX
Password: YYYYYYYYY

But it still fails with the Access Denied message.  But for some reason it works with a player like IPTV Smarters Pro (which is a massive memory hog, which is why I would like to use this)

**Reply by sebdelsol** [reply]
I don't see how I could help more without testing this service...

**Comment by sfc-juventino** [comment]
No problem. I was just wondering if you had encountered this before.
Obviously not. I have asked elsewhere without reply yet.

Thank you for your time.

**Reply by sebdelsol** [reply]
I might have found a weird bug involving Windows processes shown alive when they're actually terminated.
Anyway could you please close  _Sfvip All_ and try this in a command line[^1]:
```
cd "%AppData%/Sfvip All"
del ProxiesToRestore.json
```
[^1]: <kbd>⊞ Win</kbd>, <kbd>"cmd"</kbd> and <kbd>⏎</kbd>...

**Reply by sfc-juventino** [reply]
Same error unfortunately. Also tried with the SFVIP player and the ran the Updater which updated the player version to v1.2.7.72 and still the same (even executing your instructions after the update.

https://iptv-org.github.io/iptv/index.m3u still works but not my friends xteme account link or adding the details via username, password and server. Still works via IPTVSmartersPro

**Reply by sebdelsol** [reply]
crap :poop: there's something weird happening, still searching...

**Comment by AutoDub** [comment]
Just trying this program and having the same error. Has there been any solution? Is there another app that may work better? I just want to get away from IPTV Smarters.

**Comment by amourification** [comment]
I am getting the same error consistently these days.
